stdlib/array.ncl

```diff
@@ -70,13 +89,13 @@
         "%
       = fun f l => %map% l f,
 
-    elem_at : forall a. Num -> Array a -> a
+    at : forall a. Num -> Array a -> a
```

Comment:
Maybe just personal preference, but I think I prefer `elem_at` to just `at` here.

Reply:
Another, similar possibility: `get`.

Reply:
I like leaving it just `at`, since almost all uses will be a fully qualified `array.at` or maybe later `std.array.at`. So we're not polluting the global namespace with extremely short names and I don't think making it longer gives any more clarity.